resources/definitions/maker_made_300x.def.json

```diff
@@ -11,7 +11,7 @@
         "has_materials": true,
         "has_variants": false,
         "preferred_quality_type": "normal",
-        "has_machine_quality": false,
+        "has_machine_quality": true,

```

Comment:
Did you check if older profiles / project files still work after this change?

Reply:
Old profiles specifically for the 300x machine? The original profile would cause major issues with printing due to the default start and end gcode, so I made sure this new one works. The original 300x profile was problematic anyways.

Reply:
What Nallath meant was to see if users who added their Maker Made printer to Cura in Cura 4.10 would not get configuration errors or otherwise broken profiles due to this change. It's not about the print quality (we trust that your improvements make that better) but about ensuring that we don't break users' existing data.

Reply:
Specifically, users from previous versions will have the global quality profiles active, which are no longer valid profiles after this update.